code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -438,7 +438,7 @@
 /obj/item/healthanalyzer/AltClick(mob/user)
 	..()
 
-	if(!user.canUseTopic(src, BE_CLOSE))
+	if(!user.canUseTopic(src, BE_CLOSE) || !user.can_read(src) || user.is_blind())
```

Comment:
is this necessary? I think blind people can still toggle a machine's mode without looking.

Reply:
It's redundant.  Since you can't use the health analyzer while blind, does it matter whether they can toggle the mode or not?